ush/ufsda/misc_utils.py

```diff
@@ -32,20 +32,24 @@ def create_batch_job(job_config, working_dir, exe_path, yaml_path):
     with open(batch_script, 'w') as f:
         f.write('#!/bin/bash\n')
         if scheduler[job_config['machine']] == 'slurm':
-            f.write('#SBATCH -J GDASApp\n')
-            f.write('#SBATCH -o GDASApp.o%J\n')
-            f.write(f"#SBATCH -A {job_config['account']}\n")
-            f.write(f"#SBATCH -q {job_config['queue']}\n")
-            f.write(f"#SBATCH -p {job_config['partition']}\n")
-            f.write(f"#SBATCH --ntasks={job_config['ntasks']}\n")
-            f.write(f"#SBATCH --cpus-per-task={job_config['cpus-per-task']}\n")
-            f.write(f"#SBATCH --exclusive\n")
-            f.write(f"#SBATCH -t {job_config['walltime']}\n")
-        f.write(f"module use {job_config['modulepath']}\n")
-        f.write(f"module load GDAS/{job_config['machine']}\n")
-        f.write(f"cd {working_dir}\n")
+            sbatch = f"""#SBATCH -J GDASApp
+#SBATCH -o GDASApp.o%J
+#SBATCH -A {job_config['account']}
+#SBATCH -q {job_config['queue']}
+#SBATCH -p {job_config['partition']}
+#SBATCH --ntasks={job_config['ntasks']}
+#SBATCH --cpus-per-task={job_config['cpus-per-task']}
+#SBATCH --exclusive
+#SBATCH -t {job_config['walltime']}"""
+            f.write(sbatch)
+        commands = f"""
+module use {job_config['modulepath']}
```

Comment:
My `.bashrc` conflicts with `GDAS/orion`.   My GDASApp log file starts off with
```
Due to MODULEPATH changes, the following have been reloaded:
  1) GDAS/orion


CommandNotFoundError: Your shell has not been properly configured to use 'conda deactivate'.

```
The job successfully runs to completion but the message is disturbing.   I added `module purge` before `module use` and the warning printout went away.

Reply:
Fixed in [3640e0e](https://github.com/NOAA-EMC/GDASApp/pull/44/commits/3640e0ef2bb250f1b222561be7a3022d50b25af2)

This should've been included, `module purge` is always a good idea.

Reply:
Thanks!